### DISCUSSION TITLE
Проблема с заходом на сайты 07.08.2024 [РЕШЕНИЕ]

### DISCUSSION THREAD

**Discussion by FlyingHermes** [xvzc/SpoofDPI#112] [Show and tell]
### Шаг 1: Загрузка и распаковка GoodbyeDPI
1. Скачайте последнюю версию GoodbyeDPI с [официальной страницы релизов на GitHub](https://github.com/ValdikSS/GoodbyeDPI/releases).
2. Распакуйте архив в удобное место на вашем компьютере.

#### Firefox
1. Откройте `Настройки`.
2. Перейдите в раздел `Настройки сети`.
3. Включите опцию `Включить DNS через HTTPS`.
4. Выберите провайдера `Cloudflare`.

![Настройки DNS в Firefox](https://github.com/user-attachments/assets/13ac4ca1-59c1-4de3-aa57-410c94af8b92)

#### Google Chrome
1. Откройте `Настройки`.
2. Перейдите в раздел `Конфиденциальность и безопасность`.
3. Выберите `Использовать безопасный DNS`.
4. Включите опцию `С помощью: Cloudflare`.

### Шаг 3: Запуск GoodbyeDPI
1. Откройте командную строку (CMD) с правами администратора.
2. Перейдите в папку x86_64, где лежит .exe GoodbyeDPI.
3. Запустите GoodbyeDPI со следующими параметрами:

```sh
goodbyedpi.exe -5
```
---

**Comment by PoorMouse** [comment]
Что инстуркция для GoodbyeDPI делает в репозитории SpoofDPI?

**Reply by Gen5551** [reply]
Согласен.
